### ISSUE TITLE
Make a field mandatory

### ISSUE THREAD

**Issue by CrazyOnesHN** [hougaard/Simple-Object-Designer#297]
hi guys,

Add these fields, however, I want to make these fields mandatory, but I'm not sure what action to use BC., could you help me.

![BC1](https://github.com/hougaard/Simple-Object-Designer/assets/48975787/1fa968b1-391d-4f15-8bce-1142406b8fe3)

Best regards

**Comment by hougaard** [comment]
First, you need to find the action (post, release, something?) where the mandatory check should be performed. Then you can add the fields and the criteria.

**Comment by CrazyOnesHN** [comment]
Hi Erik,

Thanks for the feedback, I just did some tests, and it works perfect, in my case I want to add that validation when the end-user opens a new Sales Order and is adding the lines. Example, natively BC when you make a new sales order place a Red Asterisk in the "Item" field, that is the validation that I am looking to replicate, but I do not know that "Action" is since BC saves the document automatically.

**_**Erik, I also need to buy another license for "Simple object Designer", as I need to buy it and assign it to the user. **_**

Best regards

**Comment by hougaard** [comment]
Yes, the mandatory asterisk is not always the greatest because we cannot prevent saving anyway. That's why the Simple Object Designer ties the mandatory thing into actions (like posting, releasing etc...)

"Another license" -> Our webshop is always open :)
